Makefile

```diff
@@ -21,9 +21,31 @@ clean:
 	rm -rf */*/__pycache__
 	rm -rf */*/*/__pycache__
 
-install:
+.PHONY: build install
+
+build:
+	@echo "BUILDING PYTHON PACKAGE..."
+	python3 setup-partial.py build
+	@echo "BUILDING TRANSLATIONS..."
+	for LANG in `cat po/LINGUAS`; do \
+	echo $$LANG; \
+	mkdir -p build/$$LANG; \
+	msgfmt po/$$LANG.po -o build/$$LANG/catapult.mo; \
```

Comment:
I'd rather put these under a single directory, as it'd be cleaner once there's a large amount of languages, so maybe `build/mo/$$LANG.mo`?

Reply:
Sure, will do.